modules/thermal_hydraulics/doc/content/source/ics/RhoEAFromPressureTemperatureFunctionVelocityIC.md

```diff
@@ -2,10 +2,11 @@
 
 This object computes the value of $rhoEA$ given pressure and temperature as variables and velocity as a function.
 
+!alert note
+This initial condition is typically already provided by the [flow model setup](syntax/FlowModelSetup/index.md).
+
```

Comment:
```suggestion
```

Reply:
See my comment on the linked file. `FlowModelSetup` isn't anything typically used. `FlowModelSinglePhase` doesn't use it at all, for instance (probably dumb).